src/assets/i18n/es.json

```
@@ -195,7 +195,8 @@
     "TEAM": "Equipo",
     "NEWS": "Noticias",
     "SPONSORS": "Auspiciantes",
-    "WRITENEWS": "Crear Noticia"
+    "WRITENEWS": "Crear Noticia",
+    "INSTITUTIONAL": "Institucional"
```

Es una buena práctica poner una ',' al final de uno de estos elementos, ya sean arrays u objetos, ya que si agregamos una línea extra no nos muestra que hubo un cambio en la línea anterior. Hacer lo mismo en el otro archivo de traducciones 
```suggestion
    "INSTITUTIONAL": "Institucional",
```